src/main/scala/com/typesafe/sbt/SbtGit.scala

```diff
@@ -117,7 +117,19 @@ object SbtGit {
     gitDescribedVersion := gitReader.value.withGit(_.describedVersion).map(v => git.gitTagToVersionNumber.value(v).getOrElse(v)),
     gitCurrentTags := gitReader.value.withGit(_.currentTags),
     gitCurrentBranch := Option(gitReader.value.withGit(_.branch)).getOrElse(""),
-    gitUncommittedChanges in ThisBuild := gitReader.value.withGit(_.hasUncommittedChanges)
+    gitUncommittedChanges in ThisBuild := gitReader.value.withGit(_.hasUncommittedChanges),
+    scmInfo := {
+      val remote = """origin[ \t]+(?:git@|https?\:\/\/)([^:\/]+)[:\/](.*)\.git[ \t]+\(fetch\)""".r
+      
+      Process("git remote -v").lines_!.collect {
+        case remote(domain, repo) =>
+          ScmInfo(
+            url(s"https://$domain/$repo"),
+            s"scm:git:https://$domain/$repo.git",
```

Comment:
Is the `scm:git:` prefix really needed? I thought it was only the Maven release plugin which used that convention.

I would also suggest to leave out the `devConnection` field and just use the URL returned either by `git ls-remote --get-url origin` or `git remote -v` as the connection URL. For GitHub (and I guess most other hosting services) SSH and HTTPS can both be used for read/write.

Finally, avoiding to hard-code the supported URLs would be nice, but I don't know what the goal of the plugin is in terms of what Git workflows should be supported.